src/components/SideTray/Body/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import style from './style.css';
+
+const Body = ({ children, ...passedProps }) => {
+  return (
+    <div className={style.SideTray__Body} {...passedProps}>
+      {children}
```

Comment:
You don't need to enumerate children. It's just a prop, so let `passedProps` handle it.